clone/step_clone.go

```diff
@@ -26,6 +27,10 @@ func (c *CloneConfig) Prepare() []error {
 		errs = append(errs, fmt.Errorf("'linked_clone' and 'disk_size' cannot be used together"))
 	}
 
+	if c.Annotation == "" {
+		c.Annotation = "generate by jetbrains-infra/packer-builder-vsphere"
```

Comment:
`s/generate/generated/` maybe?

Reply:
I'd prefer empty. Taking a snapshot we have to set some name. Here it's not required.